README.md

```diff
@@ -169,30 +175,50 @@ The timeline can also be used to manually scrub through the video to an arbitrar
 
 Tagging and drawing regions is not possible while the video is playing.
 
-## Exporting Labels
+### Exporting Labels
 
 Once assets have been labeled, they can be exported into a variety of formats:
 
 * [Azure Custom Vision Service](https://azure.microsoft.com/en-us/services/cognitive-services/custom-vision-service/)
 * TensorFlow (Pascal VOC and TFRecords)
 * VoTT (generic JSON schema)
 
-In addition, users may choose to export all assets, only visited assets, or only tagged assets.  Click on the 'Export' icon in the left hand navigation. Select the appropriate export provider and which assets to export.
+In addition, users may choose to export
+
+* all assets
+* only visited assets
+* only tagged assets
+
+Click on the `Export` icon in the left hand navigation. Select the appropriate export provider and which assets to export.
 
 ![alt text](docs/images/export-labels.png "Export Labels")
 
-## Keyboard Shortcuts
+### Keyboard Shortcuts
```

Comment:
I would also add some of the keyboard shortcuts available within canvas tools.  Review canvas tools v1 app help for more information.  Things missing are scroll wheel and ways to resize regions with keyboard.